Calculator.rb

```diff
@@ -0,0 +1,97 @@
+# This is for the Calculator exercise, the first exercise we turn in!
+# Specifics: add, subtract, multiply and divide using two numbers
+# Silver: Print out the formula in addition to the result, i.e. 2 + 2 = 4
+  # Add support for computing exponents (2^4 = 2 * 2 * 2 * 2 = 16).
+  # Add support for the modulo operator (10 % 3 = 1).
+# Gold Gracefully handle unexpected user input:
+
+puts "Hello and welcome to Calculator."
+puts "Please specify what type of math you'd like to do:"
+
+# This is what type of calculation will be performed
+math_operation = gets.chomp.downcase
```

Comment:
This is a good variable name. You might be able to get away with just operation since it is within the context of a calculator app it may be sufficiently clear.